README.md

```diff
@@ -1,3 +1,5 @@
+![mobius-logo](https://github.com/spotify/mobius/wiki/mobius-logo.png)
```

Comment:
You should probably use `https://github.com/spotify/mobius.swift/wiki/mobius-logo.png` instead